podvm/Makefile.inc

```diff
@@ -122,6 +122,13 @@ endef
 
 binaries: $(BINARIES)
 
+# Skopeo package packages are available in RHEL/CentOS 8 or later and Ubuntu 20.10 or later
```

Comment:
I guess it does not matter of the position of the tasks. exchange `$(PAUSE)` and `$(KATA_AGENT)` is enough?

Reply:
Thanks,  I'll try to test this now